open-apparel-registry/open-apparel-registry
# Require claim form facility description

## Overview

- adjust model & add migration to make the claim a facility form
facility description a required field
- update client to indicate that facility description is a required
field on the claim a facility form
- update test for validation function

Connects #678

## Demo

![Screen Shot 2019-07-16 at 11 07 12 AM](https://user-images.githubusercontent.com/4165523/61306037-e2f4e780-a7b9-11e9-9613-04505926c631.png)

## Testing Instructions

- `./scripts/resetdb` then sign in as c1 and turn on claim a facility
- sign out then sign in to the app as c2
- search for a facility and open the claim form
- verify that the facility description claim form field is required in the UI and that you can save the form successfully

## Checklist
